src/lib/Server/Controller/ContentType.php

```diff
@@ -560,6 +560,35 @@ public function loadContentTypeFieldDefinition($contentTypeId, $fieldDefinitionI
         throw new Exceptions\NotFoundException("Field definition not found: '{$request->getPathInfo()}'.");
     }
 
+    /**
+     * @throws \EzSystems\EzPlatformRest\Exceptions\NotFoundException
+     */
+    public function loadContentTypeFieldDefinitionByIdentifier(
+        int $contentTypeId,
+        string $fieldDefinitionIdentifier,
+        Request $request
+    ): Values\RestFieldDefinition {
+        $contentType = $this->contentTypeService->loadContentType($contentTypeId, Language::ALL);
+        $fieldDefinition = $contentType->getFieldDefinition($fieldDefinitionIdentifier);
+        $path = $this->router->generate(
+            'ezpublish_rest_loadContentTypeFieldDefinitionByIdentifier',
+            [
+                'contentTypeId' => $contentType->id,
+                'fieldDefinitionIdentifier' => $fieldDefinitionIdentifier,
+            ]
+        );
+
+        if ($fieldDefinition !== null) {
+            return new Values\RestFieldDefinition(
+                $contentType,
+                $fieldDefinition,
+                $path
+            );
+        }
```

Comment:
Wouldn't be better to change this condition to `$fieldDefinition === null` and throw the exception then? Or should we expect any scenario when it should be thrown?